.clang-format

```diff
@@ -1,183 +0,0 @@
-# This file is generated by the following command
```

Comment:
Don't delete this file. It helps users format their code especially when they're using VS Code (otherwise they have to explicitly specify a "default" format style in the settings). You can make the file simpler like https://github.com/flutter-tizen/flutter-tizen/blob/master/.clang-format.